src/main.rs

```diff
@@ -104,6 +104,9 @@ fn main() {
 
     let config = if matches.is_present("ignore-config") {
         Config::with_none()
+    } else if matches.is_present("config-file") {
+        // Here we should call new method to load from custom path
+        Config::default()
```

Comment:
Yup, pretty much. If we have a config file location known by default, we can just derive from `Config::from_path` instead of default.